### ISSUE TITLE
[Bug] Ensure that fonts / layouts update in real time as you update the accessibility fonts on devices

### ISSUE THREAD

**Issue by PureWeen** [dotnet/maui#1824]

**Comment by shimingsg** [comment]
We cannot reproduce your issue with the information above, could you provide a sample project/screenshots or more details that we can reproduce your issue?

**Comment by PureWeen** [comment]
@shimingsg this is an internal .NET MAUI tracking issue don't worry about verifying

**Comment by rachelkang** [comment]
tested with .NET 8 + Recipes app - validating that this is still an issue on iOS, but not Android

**Comment by Qythyx** [comment]
I figured out why this wasn't working for me for Android. I had `ConfigChanges.FontScale` in my `ConfigurationChanges` in my main activity. When I removed this it started working. It would be nice if there was more documentation on how these flags are handled by MAUI apps.

**Comment by QianaJiao** [comment]
Verified on Visual Studio Enterprise 17.6.0 Preview 2.0. This issue not repro with sample project.
1. Turn on the VoiceOver on remote iPhone
2. Debug Maui on iOS device
3. Change some text or color on MainPage.xaml
4. Check the layout is update on device at same time and the VoiceOver reads the updated content.

Please correct me if my steps is incorrect, thank you.

**Comment by Qythyx** [comment]
@QianaJiao , I don't understand your repro. Why is VoiceOver related? I believe this issue has to do with font scaling. That is, if you launch an app, then go to iOS Settings -> Display & Brightness -> Text Size and change the Text Size, then if you switch back to your app the font sizes do not change, but they should.
